src/@types/new.ts

```diff
@@ -1,7 +1,9 @@
+import { BoxItemVersionMini } from './api';
 import { Target } from './model';
 
 export interface NewAnnotation {
     description?: NewReply;
+    file_version: Partial<BoxItemVersionMini>;
```

Comment:
Should this just be an object with an `id` property? Or even just `fileVersionId`? It doesn't seem to provide much value as partial.